src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -200,7 +204,8 @@ protected Task Stop()
                         SignatureHelpProvider = new SignatureHelpOptions
                         {
                             TriggerCharacters = new string[] { " " } // TODO: Other characters here?
-                        }
+                        },
+                        DocumentFormattingProvider = false
```

Comment:
Yes, this is part of the workaround - when I started working on this I had set it to `true`, only to run into the `FormattingOptions` issue later. So I set to to false, which is redundant as the it would be `false` by default.